test_heap.py

```diff
@@ -0,0 +1,20 @@
+# _*_ coding=utf-8 _*_
+
+HEAP_LIST = [39, 67, 23, 1, 765, 3, 2, 87]
+
+
+def test_push():
+    from binary_heap import BinHeap
+    bins = BinHeap()
+    for i in HEAP_LIST:
+        bins.push(i)
+    assert bins.heap[0] == 765
+
+
+def test_pop():
+    from binary_heap import BinHeap
+    bins = BinHeap()
+    for i in HEAP_LIST:
+        bins.push(i)
+    bins.pop()
+    assert bins.heap[0] == 87
```

Comment:
More tests. Make sure to test what happens when your heap is popping and pushing with only 1 item, with zero items, and that everything functions as expected.